README.md

```diff
@@ -1,2 +1,104 @@
-# warning
-Warning Tool for CodeX Editor
+![](https://badgen.net/badge/CodeX%20Editor/v2.0/blue)
+
+# Warning Tool
+
+Provides Warning Blocks for the [CodeX Editor](https://ifmo.su/editor).
```

Comment:
Provides Warning Block for the CodeX Editor. Block has title and message. It can be used, for example, for editorials notifications or appeals.